src/setup/platformSetup/index.desktop.js

```diff
@@ -14,6 +15,11 @@ export default function () {
         LocalNotification.showUpdateAvailableNotification();
     });
 
+    // Trigger action to show keyboard shortcuts
+    ipcRenderer.on('show-keyboard-shortcuts-modal', () => {
```

Comment:
Unnecessary arrow function:

```suggestion
    ipcRenderer.on('show-keyboard-shortcuts-modal', KeyboardShortcuts.showKeyboardShortcutModal,
```